CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ endif()
 if(${fastrtps_FOUND})
```

Comment:
I think these messages are not needed as find these packages is REQUIRED and the if can be omitted.

Reply:
I prefer to left them there to keep track of what CMake is doing